README.md

```diff
@@ -2,9 +2,7 @@
 
 FINNFlow is FINN's additions to square's flow. It's built on top of flow, and adds whatever we've needed.
 
-**Warning 1** : This is currently fairly hackish, and will remain so until square releases flow 0.9. There are some API breaking changes in there, and we're not going to clean this up until then. At that point we'll also consider PR's to get any features we need back into flow.
```

Comment:
Remove the "our flow migration" info as well, we've torn out that code long ago and it never was terribly good.